### PR TITLE
Add async/await usage example (closes #506)

### LINKED ISSUE #506: Add async/await code examples for each method
Hello! I think it's great that each method includes an example of how to use it with callbacks and Promises.

Would y'all be interested in adding an `async/await` example below the Promise example for each method as well?

### PR DESCRIPTION
**Pull request by JPeer264**
Assuming that [runkit](https://runkit.com/jpeer264/5a532978dcd17c00124b836d) works in all cases.

Should a example be added to every method as well (right under the `Promise usage:`)?

### CODE REVIEW

**Comment by RyanZim** [comment]
> Should a example be added to every method as well (right under the Promise usage:)?

I guess it couldn't hurt.

**Comment by RyanZim** [comment]
BTW, @JPeer264: I just got @jprichardson to send you an invite to be a collaborator here. Feel free to close out issues etc. For code changes other than simple docs typo fixes, we should still do internal PRs and get approval from the majority of the core team.

FYI, the core team is currently @jprichardson, @ryanzim (me), and @manidlou.

**Comment by JPeer264** [comment]
Alright I will add them in this PR as well.

Thanks a lot, I really appreciate that. Will do 👍

**Comment by JPeer264** [comment]
@RyanZim I added the async/await example everywhere where a promise example was (there was an already existent async/await example, I took this as example for everything else). I also improved the consistency.

File: docs/pathExists.md [hunk 1]
```diff
@@ -11,12 +11,23 @@ Test whether or not the given path exists by checking with the file system. Like
...
+
+// Promise usage:
+fs.pathExists(file)
+  .then(exists => console.log(exists)) // => false
+
+// With async/await:
+async function example (f) {
+  const exists = await fs.pathExists(f)
+
+  console.error(exists) // => false
```
Comment (RyanZim) [inline docs/pathExists.md]: Shouldn't this be `console.log`?

Comment (JPeer264) [reply docs/pathExists.md]: Ha, good catch!

**Pull request merged**